src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
+
```

Comment:
```suggestion
```

Reply:
Excuse me, did yopu check "solution1" branch? I dont understand where is mistakes